src/snowflake/cli/_plugins/nativeapp/codegen/compiler.py

```diff
@@ -88,7 +88,7 @@ def compile_artifacts(self):
         if not self._should_invoke_processors():
             return
 
-        with cc.phase("Invoking artifact processors"):
+        with cc.phase("Invoking artifact processors", span_name="compile_artifacts"):
```

Comment:
there's no compilation happening. How about simply "artifact_processors"? We should probably also have individual processors as sub-spans.